angrmanagement/plugins/log_reverse_engineering/log_reverse_engineering_plugin.py

```diff
@@ -0,0 +1,133 @@
+from ..base_plugin import BasePlugin
+
+try:
+    from slacrs import Slacrs
+    from slacrs.model import VariableRename, FunctionRename, ReverseEngineeringProgress
+except ImportError as ex:
+    Slacrs = None  # type: Optional[type]
+    VariableRename = None  # type: Optional[type]
+    FunctionRename = None  # type: Optional[type]
+    ReverseEngineeringProgress = None  # type: Optional[type]
+
+
+class LogReverseEngineeringPlugin(BasePlugin):
```

Comment:
You need to add a few lines to this file to prevent the plugin from being loaded if slacrs is not installed. see https://github.com/angr/angr-management/pull/288 for instructions.